### PR TITLE
Changing number of samples back to be consistent with fhe, cvr

### PR DESCRIPTION
**Pull request by ksterne**
This is more so my reminder that if change in the number of samples being passed to ACFCalculate is correct then this PR should go through.

### CODE REVIEW

File: linux/home/radar/ros.3.6/usr/codebase/superdarn/src.lib/os/site.fhw.1.0/src/site.c [hunk 1]
```diff
@@ -1076,14 +1076,14 @@ usleep(usecs);
         if (debug)
         fprintf(stderr,"FHW seq %d :: ACFCalculate acf\n",nave);
         ACFCalculate(&tsgprm,(int16 *) dest,rngoff,skpnum!=0,
-          roff,ioff,mplgs,lagtable,acfd,ACF_PART,2*total_samples,badrng,seqatten[nave]*atstp,NULL);
+          roff,ioff,mplgs,lagtable,acfd,ACF_PART,2*dprms.samples,badrng,seqatten[nave]*atstp,NULL);
```
Comment (egthomas) [inline linux/home/radar/ros.3.6/usr/codebase/superdarn/src.lib/os/site.fhw.1.0/src/site.c]: Still looking back through my emails to see if this is the correct change, but I believe there is a typo here - should be `dprm.samples` and not `dprms.samples`

Comment (ksterne) [reply linux/home/radar/ros.3.6/usr/codebase/superdarn/src.lib/os/site.fhw.1.0/src/site.c]: Whoops.  I'd found that while trying to compile but forgot to push it up to git.

**Comment by egthomas** [comment]
So just to clarify, this pull request is changing FHW back to being consistent with FHE but inconsistent with CVE/CVW.  I'll try updating the FH code on this branch to be fully consistent with its use of `dprm.samples` vs `total_samples` (in contrast to the other pull request I just opened - #4).

**Comment by ksterne** [comment]
This is obsolete with code in #4.

**Pull request closed without merging**